edge-modules/api-proxy-module/rust-sdk/azure-iot-mqtt/src/device.rs

```diff
@@ -154,6 +154,8 @@ impl futures_core::Stream for Client {
 					if *reset_session {
 						match std::pin::Pin::new(&mut this.inner).poll_next(cx) {
 							std::task::Poll::Ready(Some(Ok(mqtt3::Event::NewConnection { .. }))) => (),
+							                            
+                            std::task::Poll::Ready(Some(Ok(mqtt3::Event::Disconnected(_)))) => (),
```

Comment:
seems like formatting issue

Reply:
fixed for what I added, but for the rest of the file I didn't push the changes after I run cargo fmt because it's causing a lot of file changes.